python_modules/libraries/dagster-dbt/dagster_dbt_tests/cli/conftest.py

```diff
@@ -9,6 +9,12 @@
 @pytest.fixture(name="dbt_project_dir", scope="function")
 def dbt_project_dir_fixture(tmp_path: Path) -> Path:
     dbt_project_dir = tmp_path.joinpath("test_jaffle_shop")
-    shutil.copytree(src=test_jaffle_shop_path, dst=dbt_project_dir)
+    shutil.copytree(
+        src=test_jaffle_shop_path,
+        dst=dbt_project_dir,
+        ignore=shutil.ignore_patterns(
+            "*.wal",
```

Comment:
comment maybe

Reply:
should we avoid copying the duckdb files at all, not just the `wal`?

Reply:
eh nevermind , the one issue we hit was just from two processes trying to use `duckdb` in the source directory directly at the same time, no copy involved. 

This `wal` fix should solve the other one we saw though